devtools/mog/main_e2e_test.go

```diff
@@ -33,5 +36,27 @@ func TestE2E(t *testing.T) {
 
 	actual, err := ioutil.ReadFile(output)
 	assert.NilError(t, err)
+
+	// t.Logf("OUTPUT\n%s\n", PrependLineNumbers(string(actual)))
```

Comment:
stray debug line? Can be removed I guess

Reply:
Instead I just made it more formal: `-print-gen=true`. When I was debugging `go vet` crashes in the E2E test I found the line numbers useless, so I was commenting out the vet lines and toggling this line to print the generated code with line numbers so i could understand the vet failure that had just happened.

Now if it fails you could rerun with `-vet-gen=false -print-gen=true`